src/Output/MergeScript.php

```diff
@@ -10,12 +10,19 @@ class MergeScript implements Stringable
     public function __construct(
         protected Ziggy $ziggy,
         protected string $nonce = '',
+        protected bool $asJson = false,
     ) {}
 
     public function __toString(): string
     {
         $routes = json_encode($this->ziggy->toArray()['routes']);
 
+        if ($this->asJson) {
+            return <<<JSON
+            <script id="Ziggy_routes" type="application/json"{$this->nonce}>$routes</script>
```

Comment:
Would this actually ever need the nonce? If someone uses the asJson option but also has a nonce does the script still need the nonce, or can it be omitted since it's 'inert'?

Reply:
Answered my own question, it can be omitted. Browsers completely ignore all other attributes if they see `type="application/json"`.